files/en-us/web/css/anchor/index.md

```diff
@@ -336,11 +376,11 @@ Select different values from the drop-down menus to see how they affect the posi
 
 ### Element positioned relative to multiple anchors
 
-This example positions an element relative to two different anchors, which are used to set the position of the top-left and bottom-right corners of the anchor-positioned element. The anchors can be dragged and moved via keyboard controls, resizing the positioned element.
+This example positions an element relative to two different anchors, which are used to set the position of the top-left and bottom-right corners of the anchor-positioned element. The anchors can be moved via keyboard controls or dragged, resizing the positioned element.
```

Comment:
changed the order b/c someone it going to think we meant drag with the keyboard controls. 🤦‍♀️